review previous portuguese docs (#17)
stack-spot/graphene-java-api-stack: tracing-plugin/docs/usage.md

> ### **Use**

Reviewer: ```suggestion
### **Usage**
```

> It's possible to apply this in an application of type APP with the StackSpot CLI. Run the command below to add the plugin in your application:

Reviewer: ```suggestion
It's possible to apply this plugin in the app-type application. Run the STK CLI command to add the plugin to your application:
```

```bash
stk apply plugin graphene-java-api-stack/tracing-plugin
```

> The steps below show how to import a Stack in your local environment:

### **Requirements**

To use this plugin, you need to install the following items: 
- [**STK CLI**](https://docs.stackspot.com/v3.7.0/docs/stk-cli/installation/)
- [**Java**](https://openjdk.org/)
- [**Git**](https://git-scm.com/)

### **Configuration of STK CLI**

> Execute the command below to import/update the local catalog which contain the Graphene Java API Stack

Reviewer: ```suggestion
To import/update the local catalog with the Graphene Java API Stack, execute the command below:
```

+-------------------------+-----------------------------------------------------------+------------------+-----------------+
| name                    | description                                               | types            | version(latest) |
+-------------------------+-----------------------------------------------------------+------------------+-----------------+
| spring-web-api-template | Template for Web APIs based on Java with Spring Boot and  | ['app-template'] | no release      |
|                         | others Spring technologies.                               |                  |                 |
+-------------------------+-----------------------------------------------------------+------------------+-----------------+
```

**The command below lists all plugins locally available:**
```bash
stk list plugin
```

> **Exemple output:**

Reviewer 2: Sugestão
**Output example**

+----------------+--------------------------------------------------------------+---------+-----------------+
| metrics-plugin | This plugin adds functionalities to obtain metrics of the    | ['app'] | no release      |
|                | application using Spring Actuator component.                 |         |                 |
|                |                                                              |         |                 |
| tracing-plugin | This plugin adds functionalities to trace of the application | ['app'] | no release      |
|                | using Jaeger                                                 |         |                 |
|                |                                                              |         |                 |
| logging-plugin | This plugin gives a pre configured structure logs in JSON    | ['app'] | no release      |
|                | format.                                                      |         |                 |
+----------------+--------------------------------------------------------------+---------+-----------------+
```

### Install

> To create an application and configure all plugins related to observability, copy and paste the command below in the terminal:

Reviewer: ```suggestion
To create an application and configure all plugins related to observability, copy and paste the command below on your terminal:
```
